### PR TITLE
Deployment ready

### PR DESCRIPTION
**Pull request by anshayghosh**

### CODE REVIEW

File: src/Components/PostCarousel.js [hunk 1]
```diff
@@ -8,10 +8,25 @@ const PostCarousel = (props) => {
       <div style={{height: '100%'}}>
           <Carousel interval={null} className="candor-carousel">
             {
-              props.postsList.map( post => {
+              props.postsList.filter( post => {
```
Comment (jamiedishy) [inline src/Components/PostCarousel.js:11]: props.postsLists.filter(post => post.asset.status !== '')

File: src/Components/PostCarousel.js [hunk 2]
```diff
@@ -8,10 +8,25 @@ const PostCarousel = (props) => {
...
+              }).map( (post, index) => {
                 return (
-                  <Carousel.Item className="candor-carousel">
-                    <PostPlayer post={post} />
+                  <Carousel.Item key={index} className="candor-carousel">
+                    {
+                      post.asset.status=="deleted" ?
+                        <div >
+                          <div className="deleted-post">
+                            <h>Deleted post</h>
```
Comment (jamiedishy) [inline src/Components/PostCarousel.js:24]: Use stringlibrary

File: src/Components/GooglePlayStoreButton.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const GooglePlayStoreButton = (props) => {
```
Comment (jamiedishy) [inline src/Components/GooglePlayStoreButton.js:1]: ({width})

File: src/Components/ProfilePicture.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const ProfilePicture = (props) => {
```
Comment (jamiedishy) [inline src/Components/ProfilePicture.js:1]: const profilepic = ({photoUrl})
...
return photoUrl && (
 <img class....

**Review by jamiedishy — CHANGES_REQUESTED** [review]

File: src/Components/ProfilePicture.js (hunk 4 above)
Comment (anshayghosh) [reply src/Components/ProfilePicture.js:1]: we want to show back up as discussed on gather

**Review by jamiedishy — APPROVED** [review]

**Pull request merged**